next/components/molecules/sections/general/BannerBasicSection.tsx

```diff
@@ -0,0 +1,41 @@
+import { BannerBasicSectionFragment } from '@bratislava/strapi-sdk-homepage'
+import BannerBasic from 'components/forms/simple-components/BannerBasic'
+import { useRouter } from 'next/router'
+
+import React from 'react'
+
+type BannerBasicSectionProps = { section: BannerBasicSectionFragment }
+
+const BannerBasicSection = ({ section }: BannerBasicSectionProps) => {
+  const router = useRouter()
+
+  return (
+    <BannerBasic
+      imagePath={section.media.data?.attributes?.url}
+      content={section.content}
+      onPressPrimary={
+        section.primaryLink?.url
```

Comment:
If we would use button-styled link this would be much easier.

Reply:
Please simplify. Best to discuss in person :)